.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: go
+
+go:
+  - 1.8
+  - tip
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - go: tip
+
+sudo: required
```

Comment:
This line prevents us from using container-based builds which tend to be much faster.  Is there a reason for it?

Why not try to run it on [modern infrastructure](https://blog.travis-ci.com/2017-06-19-trusty-updates-2017-Q2) from scratch by replacing this with
```
sudo: false
dist: trusty
```